### PR TITLE
Ignore missing task I/O accounting (/proc/*/io)

### PR DESCRIPTION
**Pull request by koct9i**
This feature is missing in kernels built with CONFIG_TASK_IO_ACCOUNTING=n

It's used for I/O engine statistics: "kernel_read_bytes", "kernel_written_bytes".

### CODE REVIEW

File: yt/yt/core/misc/proc.cpp [hunk 1]
```diff
@@ -1556,21 +1556,29 @@ TTaskDiskStatistics GetSelfThreadTaskDiskStatistics()
...
-        } else if (fields[0] == "write_bytes:") {
-            TryFromString(fields[1], stat.ReadBytes);
+            if (fields[0] == "read_bytes:") {
+                TryFromString(fields[1], stat.ReadBytes);
+            } else if (fields[0] == "write_bytes:") {
+                TryFromString(fields[1], stat.ReadBytes);
+            }
+        }
+    } catch (const TSystemError& ex) {
+        if (ex.Status() == ENOENT) {
```
Comment (gritukan) [inline yt/yt/core/misc/proc.cpp:1578]: Let's log the error here.

Comment (koct9i) [reply yt/yt/core/misc/proc.cpp]: Ok.

I've added static flag to keep flood shut and warn just once.
I don't see anything like WARN_ON_ONCE() macro.

File: yt/yt/core/misc/proc.cpp [hunk 2]
```diff
@@ -1553,24 +1553,36 @@ TTaskDiskStatistics GetSelfThreadTaskDiskStatistics()
...
+                if (fields[0] == "read_bytes:") {
+                    TryFromString(fields[1], stat.ReadBytes);
+                } else if (fields[0] == "write_bytes:") {
+                    TryFromString(fields[1], stat.ReadBytes);
+                }
+            }
+        } catch (const TSystemError& ex) {
+            if (ex.Status() == ENOENT) {
+                supported = false;
+                YT_LOG_WARNING("Task I/O accounting is not supported by kernel");
```
Comment (gritukan) [inline yt/yt/core/misc/proc.cpp:1582]: YT_LOG_WARNING(ex, msg)

**Review by gritukan — APPROVED** [review]
LGTM after a small fix

**Review by Kontakter — APPROVED** [review]
Looks good

**Pull request closed without merging**